docs/lago-tutorial/README.rst

```diff
@@ -0,0 +1,223 @@
+Lago-Tutorial
+=============
+
+This repository contains the default files for creating Lago
+environment.
+
+In this tutorial you will learn how to set up a custom virtual
+environment with Lago.
+The virtual environment will be used as an infrastructure for your
+application, so you can run tests
+in order to check the functionality of your application, without the
+overhead of setting up an infrastructure.
+
+This tutorial will be “hands on”, and will take you through the process
+of creating, running and deploying the environment.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Install Lago - follow
+`this <../README.html#installation>`__ tutorial
+for more information
+
+Creating the working directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You should copy 'lago/docs/lago-tutorial' from this repository to your machine.
+
+The components  
+^^^^^^^^^^^^^^^
+
+In this directory you will find the following:
+
+-  init.json – This is the file which describes the structure of our
+   environment: the specification of the vms, networks, the path and
+   name of the deployment scripts for each vm.
+
+-  scenarios - This directory will contain default configuration files
+   for common enviornments - you are welcome
+   to conribute your on files :)
+
+-  deployment-scripts – This directory contains the deployment scripts
+   for each vm (in our case a script that will
+   install the Jenkins server)
+
+Editing init.json
+^^^^^^^^^^^^^^^^^
+
+The init.json is where we need to describe our environment e.g vms and
+networks, so Lago can create them for us.
+
+//TODO: elaborate about the different features and values that can be
+specified within the file.
+
+
+The network
+^^^^^^^^^^^
+
+Lago allows for a creation of 10 different lans ranging from:
+192.168.200.x to 192.168.209.x
+The subnet will be automatically assigend to the virtual network
+bridge.
+
+Creating the environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can use two different approches:
+
+Manual Configurations:
+
+::
+
+    lago init lago-work-dir init.json
+
+(This should be invoked from: /lago-tutorial/scenarios/jenkins)
+
+-  The directory /lago-work-dir will contain the files of our new Lago
+   environment.
+   This directory shouldn't exist before invoking lago init.
+   
+-  --template-repo-path, for specifing the path to the template-repo
+   file. (The templates repository is the place from which the base “qcow2” virtual disks will be copied from).
+   By default Lago will use the file from: http://templates.ovirt.org/repo/repo.metadata\
+   
+-  init.json, the name of the file which describes our environment.
+
+Auto configurations
```

Comment:
Better make this a subsection, you can use '++++' or whatever.

Also, you know that the '::' can be in the same line as the text right? like:

    Auto configurations::
    
        lago init